docs/architecture.md

```diff
@@ -0,0 +1,17 @@
+# Architecture
+
+Version: 1.0
+
+Note: this document is targeting the desired architecture for the first milestone (November 2022).  It will be updated as needed. 
+
+This application consists of these parts:
+* Rest Api
+  * Used for our frontend or for other applications to interact with ours
+* Postgresql Database
+  * Used for storing our data
+* Kafka Platform
+  * Used to push messages for the event listener and other applications that want information around content sources
```

Comment:
If there's an example you about what "information" would get pushed, that might be useful. But maybe it's too early for that :)